RELEASE_NOTES

```diff
@@ -29,15 +19,6 @@
    this issue by explicitly linking your application to libstdc++
    instead (e.g., mpicxx -o foo foo.cxx -lstdc++).
 
- * GCC builds - When using Homebrew or Macports gcc (not the default
-   gcc, which is just a symlink to clang), a bug in gcc
-   (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=40838) causes stack
-   alignment issues in some cases causing segmentation faults.  You
-   can workaround this issue by building mpich with clang, or by
-   passing a lower optimization level than the default (-O2).
-   Currently, this bug only seems to show up when the
-   --disable-error-checking flag is passed to configure.
```

Comment:
I don't recall we have ever fixed that. It is possible that macos has updated so it is no longer an issue.

Reply:
I tested this on `mpich-mac1` with the GCC 10 and couldn't reproduce the issue any longer. I suppose it could still be there in certain configurations, but it seems unlikely that users will run into it at this stage.